build.gradle

```diff
@@ -217,6 +218,7 @@ allprojects {
     }
     implementation group: 'com.fasterxml.jackson.module', name: 'jackson-module-jaxb-annotations', version: '2.10.3'
     implementation group: 'net.minidev', name: 'json-smart', version: '2.4.7'
+    implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-xml:2.14.2'
```

Comment:
This comment applies to line 164.  Can't add the comment there because it hasn't changed.

The ```ext['jackson.version'] = '2.14.0-rc1'``` appears to be overriding the spring boot jackson.version property.  This property was deprecated in spring boot 2.1 in favour of the ```jackson-bom.version``` property (see https://github.com/spring-projects/spring-boot/blob/2.2.x/spring-boot-project/spring-boot-dependencies/pom.xml#L85-L87) and was removed in spring boot 2.3.x.  As the line doesn't do anything and a different version of jackson is being used it should be removed.

For reference, the spring boot properties that can be overridden are listed here: https://docs.spring.io/spring-boot/docs/2.7.7/reference/htmlsingle/#appendix.dependency-versions.properties

Reply:
Will be addressed in [SDT-87](https://tools.hmcts.net/jira/browse/SDT-87)